README.md

```
@@ -1,2 +1,74 @@
+    ```bash
+    git push origin feature/mein-feature
+    ```
+5. Öffne eine Pull Request von deinem Feature-Branch in den `dev` Branch.
+
+## 📄 Lizenz
+
+Dieses Projkt unterliegt der 0BSD Lizenz. Mehr Informationen findest du in [LICENCE](/LICENCE).
+
+## Support
+
+**Issues**: https://github.com/EntchenEric/better-pruefungsplan/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)